cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -168,6 +168,7 @@ handleSimpleNode
     -- Socket directory
     myLocalAddr <- localSocketAddrInfo myNodeId (ccSocketDir cc) MkdirIfMissing
 
+    addrs <- nodeAddressInfo myNodeAddress
```

Comment:
Can this throw exceptions? Do we need to wrap the exceptions with more context info?

Reply:
It can throw an exception, but I don't think we can trigger that without failing on the `read` that parses the ip address first.